features/fixtures/rn060/android/app/build.gradle

```diff
@@ -77,7 +77,7 @@ import com.android.build.OutputFile
 
 project.ext.react = [
     entryFile: "index.js",
-    enableHermes: System.env.RN_ENABLE_HERMES == true,  // clean and rebuild if changing
+    enableHermes: !!System.env.RN_ENABLE_HERMES // clean and rebuild if changing
```

Comment:
Does this handle when `System.env.RN_ENABLE_HERMES` is `null`?

Reply:
Yes, `null` will be `false` (as will an empty string).